src/deviation.jl

```diff
@@ -82,15 +116,35 @@ function gkldiv{T<:AbstractFloat}(a::AbstractArray{T}, b::AbstractArray{T})
 end
 
 # MeanAD: mean absolute deviation
+"""
+    meanad(a::AbstractArray{T}, b::AbstractArray{T})
+
+Computes the mean absolute deviation between `a` and `b`, i.e. `mean(abs(a - b))`.
+"""
 meanad{T<:Number}(a::AbstractArray{T}, b::AbstractArray{T}) = L1dist(a, b) / length(a)
 
 # MaxAD: maximum absolute deviation
+"""
+    maxad(a::AbstractArray{T}, b::AbstractArray{T})
+
+Computes the maximum absolute deviation between `a` and `b`, i.e. `maximum(abs(a - b))`.
+"""
 maxad{T<:Number}(a::AbstractArray{T}, b::AbstractArray{T}) = Linfdist(a, b)
 
 # MSD: mean squared deviation
+"""
+    maxad(a::AbstractArray{T}, b::AbstractArray{T})
+
+Computes the mean squared deviation between a and b, i.e. mean(abs2(a - b)).
+"""
 msd{T<:Number}(a::AbstractArray{T}, b::AbstractArray{T}) = sqL2dist(a, b) / length(a)
 
 # RMSD: root mean squared deviation
+"""
+    rmsd(a::AbstractArray{T}, b::AbstractArray{T}; normalize::Bool=false)
+
+Computes the root mean squared deviation, i.e. sqrt(msd(a, b)). The keyword argument normalize is default to false. If it is set to true, the result is normalized by (maximum(a) - minimum(a).
```

Comment:
"The keyword argument normalize is default to false." is redundant with the signature. Just say:
```
If `normalize` is `true`, the result is divided by `maximum(a) - minimum(a)`.
```